package.json

```diff
@@ -28,12 +28,12 @@
     "@angular/cdk": "^14.2.7",
     "@angular/common": "14.2.12",
     "@angular/compiler": "14.2.12",
-    "@angular/core": "^14.2.12",
+    "@angular/core": "^14.3.0",
```

Comment:
Question:
Why were `core` & `router` updated to `14.3.0`?

Reply:
@ankypant This is a minor version and its okii to put it to next minor as there is already a caret and also the migration guide for the ui-ang 9 had this angular version.
https://backbase.io/developers/angular/design-system/ui-ang/9.0.0/additional-documentation/migration-notes.html

Reply:
Agree with a caret, but it should be in place for all angular packages then. Currently there is an issue with `peerDependencies`:

```
npm view @angular/router@14.3.0 peerDependencies
{
  '@angular/core': '14.3.0',
  '@angular/common': '14.3.0',
  '@angular/platform-browser': '14.3.0',
  rxjs: '^6.5.3 || ^7.4.0'
}
```

As you can see, `@angular/router` depends on `@angular/common` 14.3.0, but in the `package.json` we have 14.2.12 without a caret.